radicle-httpd/src/api/v1/projects.rs

```diff
@@ -1454,4 +1531,443 @@ mod routes {
+
+        let response = get(
+            &app,
+            format!("/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/patches/{PATCH_ID}"),
+        )
+        .await;
+
+        assert_eq!(
+            response.json().await,
+            json!({
+              "id": PATCH_ID,
+              "author": {
+                  "id": "did:key:z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi",
+              },
+              "title": "A new `hello word`",
+              "description": "change `hello world` in README to something else",
+              "state": { "status": "proposed" },
+              "target": "delegates",
+              "tags": [],
+              "revisions": [
+                {
+                  "id": "6ec9a764a888576abc7e582dbf82a31e23a9789d",
+                  "description": "",
+                  "base": "f604ce9fd5b7cc77b7609beda45ea8760bee78f7",
+                  "oid": "1e978d19f251cd9821d9d9a76d1bd436bf0690d5",
+                  "merges": [],
+                  "discussions": [],
+                  "timestamp": 1671125284,
+                  "reviews": [],
+                },
+                {
+                  "id": "bc703aece4ac8c94836b13ad5783a8326f2b70f1",
+                  "description": "This is a new revision",
+                  "base": "f604ce9fd5b7cc77b7609beda45ea8760bee78f7",
+                  "oid": "1e978d19f251cd9821d9d9a76d1bd436bf0690d5",
+                  "merges": [],
+                  "discussions": [],
+                  "timestamp": 1671125284,
+                  "reviews": [],
+                }
+              ],
+            })
+        );
+    }
+
+    #[tokio::test]
+    async fn test_projects_patches_edit() {
+        let tmp = tempfile::tempdir().unwrap();
+        let ctx = test::seed(tmp.path());
+        let app = super::router(ctx.to_owned());
+        test::create_session(ctx).await;
+        let body = serde_json::to_vec(&json!({
+          "type": "edit",
+          "title": "This is a updated title",
+          "description": "Lets write some description",
+          "target": "delegates",
+        }))
+        .unwrap();
+        let response = patch(
+            &app,
+            format!("/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/patches/{PATCH_ID}"),
+            Some(Body::from(body)),
+            Some("u9MGAkkfkMOv0uDDB2WeUHBT7HbsO2Dy".to_string()),
+        )
+        .await;
+
+        assert_eq!(response.status(), StatusCode::OK);
+
+        let response = get(
+            &app,
+            format!("/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/patches/{PATCH_ID}"),
+        )
+        .await;
+
+        assert_eq!(
+            response.json().await,
+            json!({
+              "id": PATCH_ID,
+              "author": {
+                  "id": "did:key:z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi",
+              },
+              "title": "This is a updated title",
+              "description": "Lets write some description",
```

Comment:
```suggestion
              "description": "Let's write some description",
```
I know it's just a test, but still :)
There's a few other places with this error.